bin/post_compile

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
```

Comment:
any advantage to this over:

```
release: python manage.py migrate
```

in the Procfile?

Reply:
The release command is newer, I'll move over to this instead of the experimental python post_compile and rebase.